--- qemuctl0.2/Makefile~	2007-04-03 10:20:07.000000000 +0200
+++ qemuctl0.2/Makefile	2007-04-03 10:23:02.000000000 +0200
@@ -2,13 +2,14 @@
 
 ## Config
 # Edit these to fit your system:
+PKGNAME=qemuctl
 VERSION=0.2
-PREFIX=/usr
+PREFIX=/usr/local
 MANDIR=$(PREFIX)/share/man/man1
 BINDIR=$(PREFIX)/bin
-SHAREDIR=$(PREFIX)/share/qemuctl
+SHAREDIR=$(PREFIX)/share/$(PKGNAME)
 PIXMAPSDIR=$(PREFIX)/share/pixmaps
-DOCSDIR=$(PREFIX)/share/doc/qemu-launcher
+DOCSDIR=$(PREFIX)/share/doc/$(PKGNAME)
 LOCALEDIR=$(PREFIX)/share/locale
 APPLICATIONSDIR=$(PREFIX)/share/applications
 ## End config
